docs/earth/xdggs.md

```diff
@@ -0,0 +1,59 @@
+---
+jupytext:
+  text_representation:
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  name: python
+---
+
+# xdggs: DGGSIndex
+
+````{grid}
+```{grid-item}
+:columns: 3
+```{image} https://xdggs.readthedocs.io/en/latest/_static/logos/xdggs_logo.png
+---
+alt: xdggs logo
+width: 200px
+align: center
+---
+```
+```{grid-item}
+:columns: 9
+```{seealso}
+Learn more at the [xdggs](https://xdggs.readthedocs.io/en/latest/) documentation page.
+```
+````
+
+## Highlights
+
+xdggs provides a unified API for interacting with various Discrete Global Grid Systems (DGGS). It does so by parsing out grid metadata from the spatial coordinate containing the cell ids and persisting it on a "meta-index", an index that wraps another index (most commonly a `pandas` index).
```

Comment:
Some cool grid images would be nice here. Also we'll need to update `index.md` other wise it won't render

Reply:
I don't have a good image for healpix right now (other than [this one](https://healpix.sourceforge.io/html/introf1.png)), so for now I'll put H3 there instead.